### PR TITLE
Created a section on the docs homepage for the implementation guides. 

### PR DESCRIPTION
**Pull request by sethijs**
Added the following implementation guides: 

- Thales CipherTrust Transparent Encryption
- Liquibase Pro
- Nutanix AHV
- Swarm64 DA
- Repostor Data Protector for PostgreSQL

### CODE REVIEW

File: advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 | &nbsp;     | &nbsp; |
 | ----------- | ----------- |
 | **Partner Name**      | Thales       |
 | **Partner Product**   |  CipherTrust Transparent Encryption       |
 | **Web Site**      |    https://cpl.thalesgroup.com/encryption    |
-| **Version & Platform**   |  7.1.0, Available platforms: Windows , Linux      |
-| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management privileged user access control and detailed data access audit logging. This protects data wherever it resides, on- premises, across multiple clouds and within big data, and container environments.   |
\ No newline at end of file
+| **Version & Platform**   |  7.1.0, Available platforms: Windows, Linux      |
+| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management, privileged user access control and detailed data access audit logging. This protects data wherever it resides, on-premises, across multiple clouds and within big data, and container environments.   |
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx:13]:
```suggestion
| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management, privileged user access control, and detailed data access audit logging. This protects data wherever it resides: on-premises, across multiple clouds and within big data, and container environments.   |
```

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 2]
```diff
@@ -1,8 +1,15 @@
 ---
-title: 'Solution Summary'
+title: '2. Solution Summary'
 description: 'A brief explanation of the solution and its purpose'
 ---
-Thales CipherTrust Transparent Encryption (CTE) is designed to meet data security compliance and best practice requirements with minimal disruption. The CTE agents are installed at the operating file-system or device layer, and encryption and decryption is transparent to all applications that run above it. CTE agents are installed on EDB Postgres Advanced and Postgres Extended database servers and protect the database directories. The solution works in conjunction with the FIPS 140-2 up to Level 3 compliant CipherTrust Manager, which centralizes encryption key and policy management for the CipherTrust Data Security Platform.
+
+
+Thales’ CipherTrust Transparent Encryption secures data at-rest for EDB Postgres databases and backups
+with file system-level encryption backed by centralized key management, privileged user access controls and
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:8]:
```suggestion
with file system-level encryption backed by centralized key management, privileged user access controls, and
```

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+detailed data access audit logging. CipherTrust Transparent Encryption allows customers to adopt EDB Postgres
+for highly-sensitive and regulated data both on-premises and in the cloud while also meeting their compliance
+obligations. CipherTrust Transparent Encryption has been certified with EDB Postgres Advanced standalone,
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:11]:
```suggestion
obligations. CipherTrust Transparent Encryption has been certified with EDB Postgres Advanced ,
```

Comment (drothery-edb) [reply advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:11]: Not sure if you need "standalone" here but it is Scott's call.

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+and with EDB Postgres Extended as part of a BDR (bi-directional replication) cluster, and with barman.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:12]:
```suggestion
EDB Postgres Extended as part of a BDR (bi-directional replication) cluster, and Barman.
```

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 5]
```diff
@@ -1,8 +1,15 @@
 ---
-title: 'Solution Summary'
+title: '2. Solution Summary'
 description: 'A brief explanation of the solution and its purpose'
 ---
-Thales CipherTrust Transparent Encryption (CTE) is designed to meet data security compliance and best practice requirements with minimal disruption. The CTE agents are installed at the operating file-system or device layer, and encryption and decryption is transparent to all applications that run above it. CTE agents are installed on EDB Postgres Advanced and Postgres Extended database servers and protect the database directories. The solution works in conjunction with the FIPS 140-2 up to Level 3 compliant CipherTrust Manager, which centralizes encryption key and policy management for the CipherTrust Data Security Platform.
+
+
+Thales’ CipherTrust Transparent Encryption secures data at-rest for EDB Postgres databases and backups
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:7]:
```suggestion
Thales’ CipherTrust Transparent Encryption secures data at-rest for Postgres databases and backups
```

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 6]
```diff
@@ -1,8 +1,15 @@
...
+with file system-level encryption backed by centralized key management, privileged user access controls and
+detailed data access audit logging. CipherTrust Transparent Encryption allows customers to adopt EDB Postgres
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:9]:
```suggestion
detailed data access audit logging. CipherTrust Transparent Encryption allows customers to adopt Postgres
```

File: advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx [hunk 7]
```diff
@@ -1,12 +1,13 @@
 ---
-title: 'Partner Information'
+title: '1. Partner Information'
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx:2]: Remove the numbers from the titles in all the files. Numbering isn't necessary in online content and numbering is automatically added to the PDFs, I believe).

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx [hunk 8]
```diff
@@ -11,4 +18,4 @@ Thales CipherTrust Transparent Encryption (CTE) is designed to meet data securit
 
 
 !!! Note
-    EDB Postgres Extended with BDR (Bi-Directional Replication)
\ No newline at end of file
+    EDB Postgres Extended represents EDB Postgres Extended* with BDR (Bi-Directional Replication) and barman.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:21]: @josh-heyer I think we need to include a closing !!! on the line following the text of this note. The note is being interpreted as a heading in the PDF.

File: advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx (hunk 1 above)
Comment (drothery-edb) [reply advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx:13]: I would introduce the short name at first instance of the full product name. Then you can use the short name going forward.

File: advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx (hunk 8 above)
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx:21]:
```suggestion
    EDB Postgres Extended represents EDB Postgres Extended* with BDR (Bi-Directional Replication) and Barman.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 9]
```diff
@@ -1,24 +1,22 @@
...
-title: 'Implementing CipherTrust Transparent Encryption (CTE)'
+title: '3. Implementing CipherTrust Transparent Encryption (CTE)'
 description: 'A walk-through of setting up CTE'
 ---
 
-Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:
+**Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:**
 
-1. Postgres Server installed and in operation.
+  1. Postgres Server installed and operational.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:8]: Make this a bulleted list unless there is some order implied.

Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:8]: Postgres Server is a little ambiguous considering the three flavors of Postgres and that one of them was traditionally called Postgres Advanced Server. I think we may want to refer to server generically by using lower case, if what we mean is the Postgres server for whatever flavor of Postgres you are using.

Comment (drothery-edb) [reply advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:8]: That said, I'm not sure you need to talk about the components here as they are shown in the previous architecture diagram and discussed in the following Prerequisites section.

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 10]
```diff
@@ -1,24 +1,22 @@
...
+**The following diagram shows the basic flow of the CTE solution:**
 
 <p align="center">
   <img  src="Images/ImplementingCTE.png">
 </p>
 
-### Prerequisites
+### 3.1 Prerequisites
 #### Postgres Host
 1. Ensure that the Postgres Server is installed and running.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:21]: These don't need to be numbered steps. Change Postgres Server to Postgres server.

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 11]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
 
-### Configuring CipherTrust Manager
+### 3.2 Configuring CipherTrust Manager
 Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
 
-1. **Create  Registration Token**
+1. Create  Registration Token
 
-    a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
+  a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:41]: @josh-heyer, this sub-step is not wrapping correctly. Note sure what the issue is.

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 12]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
   <img  src="Images/CipherTrustManager.png">
 </p>
 
-### Configuring CipherTrust Manager
+### 3.2 Configuring CipherTrust Manager
 Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
 
-1. **Create  Registration Token**
+1. Create  Registration Token
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:39]:
```suggestion
1. Create a registration token.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx (hunk 11 above)
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:41]:
```suggestion
  a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token is used for the CTE agent enrollment to CM. 
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 13]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
 
-    a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
+  a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
 
-    b. Select **New Registration Token** to create a new Registration Token. The following screenshot shows a Registration Token created with the name **edb**.
+  b. Select **New Registration Token** to create a new Registration Token.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:43]:
```suggestion
  b. Select **New Registration Token** to create a new registration token. 
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 14]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
+1. Create  Registration Token
 
-    a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
+  a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
 
-    b. Select **New Registration Token** to create a new Registration Token. The following screenshot shows a Registration Token created with the name **edb**.
+  b. Select **New Registration Token** to create a new Registration Token. 
+  
+
+**The following screenshot shows a Registration Token created with the name edb**.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:46]:
```suggestion
The following screenshot shows a registration token created with the name **edb**.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 15]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
-2. **Create User Sets**
+2. Create User Sets
 
-    a. Navigate to **CTE** and select **Policies, Policy Elements** and then **User Sets**. 
+  a. Navigate to CTE and select Policies, Policy Elements and then User Sets. 
 
-    b. Select **Create User Set** to create a new User Set. The following screenshots show the User Sets created, **Postgres, EnterpriseDB and Barman**.
+  b. Select Create User Set to create a new User Set. 
+    
+**The following screenshots show the User Sets created, Postgres, EnterpriseDB and Barman.**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:59]:
```suggestion
The following screenshots show the user sets created, Postgres, EnterpriseDB and Barman.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 16]
```diff
@@ -70,7 +73,8 @@ Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
 
     a. Navigate back to **Policies** and select **Create Policy**.
 
-    b. The following screenshots show Live Data Transformation (LDT) policies **postgres-policy, epas-policy and barman-policy**. 
+    
+**The following screenshots show Live Data Transformation (LDT) policies postgres-policy, epas-policy and barman-policy.**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:77]:
```suggestion
The following screenshots show postgres-policy, epas-policy, and barman-policy Live Data Transformation (LDT) policies.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 17]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
+
+**The following screenshot shows a Registration Token created with the name edb**.
 
 
 <p align="center">
   <img  src="Images/ConfiguringCipherTrustManager.png">
 </p>
 
-2. **Create User Sets**
+2. Create User Sets
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:53]:
```suggestion
2. Create user sets.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 18]
```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
...
 
-    a. Navigate to **CTE** and select **Policies, Policy Elements** and then **User Sets**. 
+  a. Navigate to CTE and select Policies, Policy Elements and then User Sets. 
 
-    b. Select **Create User Set** to create a new User Set. The following screenshots show the User Sets created, **Postgres, EnterpriseDB and Barman**.
+  b. Select Create User Set to create a new User Set.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:57]:
```suggestion
  b. Select Create User Set to create a new user set. 
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 19]
```diff
@@ -70,7 +73,8 @@ Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
 
     a. Navigate back to **Policies** and select **Create Policy**.
 
-    b. The following screenshots show Live Data Transformation (LDT) policies **postgres-policy, epas-policy and barman-policy**. 
+    
+**The following screenshots show Live Data Transformation (LDT) policies postgres-policy, epas-policy and barman-policy.**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:78]: I wouldn't bold the numbered step and make it a sentence instead.

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 20]
```diff
@@ -70,7 +73,8 @@ Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
...
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:79]: I don't quite follow the note or understand why it ends in a colon.

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 21]
```diff
@@ -99,12 +103,14 @@ Refer to the following guides from Thales for installing the CTE agent on the Po
...
 
 !!! Note
-    You will need the **Registration Token** and host address of the **CipherTrust Manager** during the installation.
+    You will need the Registration Token and host address of the CipherTrust Manager during the installation.
 
 After the CTE agent is successfully installed, verify the Postgres host is registered with CM.
 1. Log on to the CM Web GUI and navigate to **CTE**.
 2. Select **Clients**. The client status should appear as **Healthy** as shown below (you may have to wait a few seconds for the status to get updated).
 
+**The following screenshot shows clients registered with the CipherTrust Manager.**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:112]:
```suggestion
The following screenshot shows clients registered with the CipherTrust Manager.
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 22]
```diff
@@ -99,12 +103,14 @@ Refer to the following guides from Thales for installing the CTE agent on the Po
 [CTE Agent Advanced Installation Guide](https://thalesdocs.com/ctp/cte/Books/Online-Files/7.0.0/CTE_Agent_Linux_Adv_Config_Integration_Guide_v7.0.0_Doc_v6.pdf)
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:103]:
```suggestion
[*CTE Agent Advanced Installation Guide*](https://thalesdocs.com/ctp/cte/Books/Online-Files/7.0.0/CTE_Agent_Linux_Adv_Config_Integration_Guide_v7.0.0_Doc_v6.pdf)
```

File: advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx [hunk 23]
```diff
@@ -99,12 +103,14 @@ Refer to the following guides from Thales for installing the CTE agent on the Po
...
 
 !!! Note
-    You will need the **Registration Token** and host address of the **CipherTrust Manager** during the installation.
+    You will need the Registration Token and host address of the CipherTrust Manager during the installation.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx:106]:
```suggestion
    You will need the registration token and host address of the CipherTrust Manager during the installation.
```

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 24]
```diff
@@ -1,25 +1,28 @@
 ---
-title: 'Using CipherTrust Transparent Encryption (CTE)'
+title: '4. Using CipherTrust Transparent Encryption (CTE)'
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:2]: No need to surround Postgres in parentheses 
No need for the comma before "and"

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 25]
```diff
@@ -1,25 +1,28 @@
...
+**The following diagram shows the CTE architecture.**
+
 <p align="center">
   <img  src="Images/UsingCTE.png">
 </p>
 
-### Sample User Scenarios
+### 4.1 Sample User Scenarios
 
 This section describes sample user scenarios of deploying CTE solutions on Postgres hosts such as
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:16]:
```suggestion
This section describes sample user scenarios of deploying CTE solutions on EDB Postgres Advanced Server and EDB Postgres Extended with BDR hosts.
```

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 26]
```diff
@@ -1,25 +1,28 @@
...
   <img  src="Images/UsingCTE.png">
 </p>
 
-### Sample User Scenarios
+### 4.1 Sample User Scenarios
 
 This section describes sample user scenarios of deploying CTE solutions on Postgres hosts such as 
-- EDB Postgres Advanced Server
-- EDB Postgres Extended with BDR
+- **EDB Postgres Advanced Server**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:17]: No need for the bulleted list

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 27]
```diff
@@ -1,25 +1,28 @@
...
+- **EDB Postgres Extended with BDR**
 
-**EDB Postgres Advanced Server 13 (Single Instance)**
+**EDB Postgres Advanced Server (Single Instance)**
 
 1. Install CTE agent on the Postgres host. 
 2. Login to the Postgres host and stop the postgres server.
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:23]: Capitalize Postres whenever referring to Postgres server

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 28]
```diff
@@ -63,10 +66,10 @@ The following diagram shows the BDR-Always-ON architecture. For more details, re
 
 11. Restart the Postgres server on the Lead Master node as the user `postgres`.  Make sure you are logged in using ssh (not sudo).
 
-12. The following screenshot shows a GuardPoint created for Lead Master in data center A.
+**The following screenshot shows a GuardPoint created for Lead Master in data center A.**
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:69]:
```suggestion
The following screenshot shows a GuardPoint created for Lead Master in data center A.
```

File: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx [hunk 29]
```diff
@@ -63,10 +66,10 @@ The following diagram shows the BDR-Always-ON architecture. For more details, re
...
 
 <p align="center">
   <img  src="Images/SampleUserScenarios4.png">
 </p>
 
-13. Repeat steps 2 through 11 for postgres and barman nodes in data center B (DC B).
\ No newline at end of file
+12. Repeat steps 2 through 11 for postgres and barman nodes in data center B (DC B).
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx:75]:
```suggestion
12. Repeat steps 2 through 11 for Postgres and Barman nodes in data center B (DC B).
```

File: advocacy_docs/partner_docs/ThalesGuide/07-Appendix.mdx [hunk 30]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Appendix'
+title: '6. Appendix'
 description: 'Utilize the config.yml file below'
```
Comment (drothery-edb) [inline advocacy_docs/partner_docs/ThalesGuide/07-Appendix.mdx:3]: do we want real names and IP addresses in the sample? I would think not.

**Comment by josh-heyer** [comment]
@sethijs and @drothery-edb - I've rebased this (along with some minor cleanup) to resolve merge conflicts. If you've previously pulled locally, your branch will need to be refreshed; either `git reset --hard future/productguides` (if you've no local changes or commits) or `git pull --rebase` (if you do).

cc @jericson - wanna get this through review so we're not going down to the wire for Jasmer here :)

**Pull request closed without merging**